pages/Enriched-Nether-Ice.md

```diff
@@ -0,0 +1,7 @@
+Enriched Nether Ice is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used to create [Nether Ice Coolant Cells](https://github.com/Slimefun/Slimefun4/wiki/Nether-Ice-Coolant-Cell), a powerful reactor coolant.
+
+## Obtaining
+Enriched Nether Ice is crafted with a [Heated Pressure Chamber](https://github.com/Slimefun/Slimefun4/wiki/Heated-Pressure-Chamber) using [1 Plutonium](https://github.com/Slimefun/Slimefun4/wiki/Plutonium) and [1 Nether Ice](https://github.com/Slimefun/Slimefun4/wiki/Nether-Ice).
```

Comment:
```suggestion
Enriched Nether Ice is crafted with a [Heated Pressure Chamber](https://github.com/Slimefun/Slimefun4/wiki/Heated-Pressure-Chamber) using 1 [Plutonium](https://github.com/Slimefun/Slimefun4/wiki/Plutonium) and 1[Nether Ice](https://github.com/Slimefun/Slimefun4/wiki/Nether-Ice).
```

Reply:
You missed a space, so I'm marking this as resolved and manually changing it.